openedx/core/djangoapps/schedules/management/commands/__init__.py

```diff
@@ -40,8 +40,8 @@ def add_arguments(self, parser):
     def handle(self, *args, **options):
         self.log_debug('Args = %r', options)
 
-        if 'weeks' in options:
-            num_weeks = options['weeks']
+        num_weeks = options.get('weeks', None)
```

Comment:
Don't technically need the `None` default, right? That's already the default setting for .get

Reply:
Yeah, good point.